hubble/Makefile

```diff
@@ -47,16 +51,16 @@ local-release: clean
 		for ARCH in $$ARCHS; do \
 			echo Building release binary for $$OS/$$ARCH...; \
 			test -d release/$$OS/$$ARCH|| mkdir -p release/$$OS/$$ARCH; \
-			$(MAKE) hubble GOOS=$$OS GOARCH=$$ARCH EXT=$$EXT TARGET_DIR=release/$$OS/$$ARCH; \
+			$(MAKE) $(TARGET) GOOS=$$OS GOARCH=$$ARCH EXT=$$EXT TARGET_DIR=release/$$OS/$$ARCH; \
 			tar -czf release/$(TARGET)-$$OS-$$ARCH.tar.gz -C release/$$OS/$$ARCH $(TARGET)$$EXT; \
 			(cd release && sha256sum $(TARGET)-$$OS-$$ARCH.tar.gz > $(TARGET)-$$OS-$$ARCH.tar.gz.sha256sum); \
 		done; \
 		rm -r release/$$OS; \
 	done;
 
-
-
 clean:
 	rm -f $(TARGET)
 
-.PHONY: all hubble release
+
+# Keep this include at the end of the file
+-include Makefile.override
```

Comment:
This should not be at the end of the file for `hubble/Makefile` as the override is used on the variables. Please keep all of these in their original places.

Reply:
Thanks, updated 👍